src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx

```diff
@@ -260,6 +258,51 @@ version.
 
 *When package is not specified, support is within the `newrelic` package.
 
+## AI Monitoring Support
+
+The Node.js agent supports the following AI platforms and integrations.
+
+### Amazon Bedrock
+
+Through the `@aws-sdk/client-bedrock-runtime` module, we support:
+
+| Model | Image | Text |
+| --- | --- | --- |
+| Anthropic Claude | ❌ | ✅ |
+| Cohere | ❌ | ✅ |
+| AI21 Labs Jurassic-2 | ❌ | ✅ |
+| Meta Llama2 | ❌ | ✅ |
+| Amazon Titan | ❌ | ✅ |
+
+Note: if a model supports streaming, we also instrument the streaming variant.
+
+
+### Langchain
+
+The following general features of Langchain are supported:
+
+| Agents | Chains | Tools | Vectorstores |
+| --- | --- | --- | --- |
+| ✅ | ✅ | ✅ | ✅ |
+
+Models/providers are generally supported transitively by our instrumentation of the provider's module.
+
+| Provider | Supported | Transitively |
+| --- | --- | --- |
+| Azure OpenAI | ❌ | ❌ |
+| Amazon Bedrock | ❌ | ❌ |
+| OpenAI | ✅ | ✅ |
+
+
+### OpenAI
+
+Through the `openai` module, we support:
+
+| Audio | Chat | Completions | Embeddings | Files | Images |
+| --- | --- | --- | --- | --- | --- |
+| ❌ | ✅ | ✅ | ✅ | ❌ | ❌ |
+
```

Comment:
Docs team: I want to call out that this is a new section we added to address a miss in our release of AI Monitoring. This section details all of the AI Monitoring features our Node.js agent supports.